astro/public/js/ScrollHighlight-0.0.1.js

```diff
@@ -0,0 +1,39 @@
+document.addEventListener("DOMContentLoaded", function() {
+  const tocLinks = document.querySelectorAll("nav a");
```

Comment:
This selector is really broad and breaks the section nav highlighting in `/docs/sdks` at least, and possibly other places.

Instead of inserting these scripts in the Head like this I think we should add a script tag near where the table of contents is implemented and use classes or data attributes to specifically target the elements that we want to highlight as that way it will be clear exactly what we are looking to affect

https://docs.astro.build/en/guides/client-side-scripts/